packages/patternfly-4/react-charts/src/components/ChartArea/examples/ChartArea.md

```diff
@@ -225,6 +228,7 @@ class MultiColorChart extends React.Component {
 ```
 
 ## Sparkline chart
+
```

Comment:
Can you please remove the extra lines? I've made changes to this file, over multiple PRs, and wish to avoid merge conflicts

Reply:
sure, I'll remove them! sorry -- added those to resolve the linting errors I was getting